models/models.js

```diff
@@ -7,6 +7,33 @@ exports.selectCategories = () => {
         })
 }
 
+exports.selectReviews = (query) => {
+    const validCategories = ["euro game", "social deduction", "dexterity", "children's games"]
```

Comment:
This is going to break in production as our test data may have different categories/less categories than the dev/production data. We need to fetch this information from the database at the time of request to make sure that if a category is added or deleted later, we dont have to rewrite this code